docs/fhir-web-docker-deployment.md

```diff
@@ -0,0 +1,210 @@
+        - 'EXPRESS_OPENSRP_CLIENT_ID=<keycloak-client-id>'
+        - 'EXPRESS_OPENSRP_CLIENT_SECRET=<keycloak-client-secret>'
+        - 'EXPRESS_OPENSRP_ACCESS_TOKEN_URL=<keycloak-domain>/auth/realms/<keycloak-realm>/protocol/openid-connect/token'
+        - 'EXPRESS_OPENSRP_AUTHORIZATION_URL=<keycloak-domain>/auth/realms/<keycloak-realm>/protocol/openid-connect/auth'
+        - 'EXPRESS_KEYCLOAK_LOGOUT_URL=<keycloak-domain>/auth/realms/<keycloak-realm>/protocol/openid-connect/logout'
+        - 'EXPRESS_OPENSRP_USER_URL=<keycloak-domain>/auth/realms/<keycloak-realm>/protocol/openid-connect/userinfo'
+
+        # opensrp server
+        - 'EXPRESS_OPENSRP_LOGOUT_URL=null'
+
+        # fhir web
+        - 'EXPRESS_OPENSRP_CALLBACK_URL=<fhir-web-domain>/oauth/callback/OpenSRP/'
+        - 'EXPRESS_FRONTEND_OPENSRP_CALLBACK_URL=<fhir-web-domain>/fe/oauth/callback/opensrp'
+        - 'EXPRESS_SERVER_LOGOUT_URL=<fhir-web-domain>/logout'
+
+        # UUID's
+        - 'EXPRESS_SESSION_SECRET=<randomly-generated-secret-string>'
+
+        # content security policy configuration
+        # remove "<optional-sentry-domain>" config if your deployment has no sentry
+        - 'EXPRESS_CONTENT_SECURITY_POLICY_CONFIG={"connect-src":["''self''","<optional-sentry-domain>","<keycloak-domain>","<fhir-server-domain>","<fhir-server-domain>"],"default-src":["''self''"],"img-src":["''self''","https://github.com/opensrp/","https://*.githubusercontent.com/opensrp/"],"script-src":["''self''","''unsafe-inline''"]}'
+
+      # optional sentry config
+      # - 'EXPRESS_RESPONSE_HEADERS={"report-to":", {endpoints:[{url:https://<optional-sentry-domain>/api/<optional-sentry-projectId>/security/?sentry_key=<optional-sentry-key>\\u0026sentry_environment=<optional-sentry-environment>\\u0026sentry_release=<optional-sentry-release-name>}],group:csp-endpoint,max_age:10886400}"}'
+
+      # optional redis and redis sentinel session store config (use either or neither not both)
+      # - 'EXPRESS_REDIS_STAND_ALONE_URL=redis://username:password@redis-domain:port/db'
+      # - 'EXPRESS_REDIS_SENTINEL_CONFIG={"name":"sentinelMasterName","sentinelPassword":"sentinelMasterPassword","sentinels":[{"host":"sentinel-node-1-domain","port":"12345"},{"host":"sentinel-node-2-domain","port":"12345"},{"host":"sentinel-node-3-domain","port":"12345"}]}'
+    ```
+
+    ```js
+    // config.js.tpl
+
+    window._env_ = {
+      // keycloak
+      REACT_APP_KEYCLOAK_API_BASE_URL: '<keycloak-domain>/auth/admin/realms/<keycloak-realm>',
+      REACT_APP_KEYCLOAK_LOGOUT_URL:
+        '<keycloak-domain>/auth/realms/<keycloak-realm>/protocol/openid-connect/logout',
+      REACT_APP_OPENSRP_ACCESS_TOKEN_URL:
+        '<keycloak-domain>/auth/realms/<keycloak-realm>/protocol/openid-connect/token',
+      REACT_APP_OPENSRP_AUTHORIZATION_URL:
+        '<keycloak-domain>/auth/realms/<keycloak-realm>/protocol/openid-connect/auth',
+      REACT_APP_OPENSRP_CLIENT_ID: '<keycloak-client-id>',
```

Comment:
@p-netm what do you have in mind? I was thinking this could replace the [dockerhub documentation](https://github.com/opensrp/web/blob/master/docs/how-to-use-docker-image.md) file?

Reply:
PS: I've stripped off all old opensrp envs to only leave FHIR web config envs

Reply:
decision made: is to merge these 2 files: this and the current existing docker manual.